<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DocumentInfoDialog</class>
    <widget class="QDialog" name="DocumentInfoDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>500</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Document info</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="font">
                                <font>
                                    <pointsize>9</pointsize>
                                    <weight>75</weight>
                                    <bold>true</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string>Document name:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="docNameLineEdit"/>
                    </item>
                    <item>
                        <spacer name="verticalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>0</width>
                                    <height>10</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="font">
                                <font>
                                    <pointsize>9</pointsize>
                                    <weight>75</weight>
                                    <bold>true</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string>Collaborators:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QListView" name="listView"/>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <property name="sizeConstraint">
                                <enum>QLayout::SetDefaultConstraint</enum>
                            </property>
                            <item>
                                <widget class="QPushButton" name="addCPushButton">
                                    <property name="text">
                                        <string>Add collaborator</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="removeCPushButton">
                                    <property name="text">
                                        <string>Remove collaborator</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="generateURL">
                                    <property name="text">
                                        <string>Generate invite link</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="verticalSpacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>0</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QDialogButtonBox" name="buttonBox">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="standardButtons">
                                <!--set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set-->
								<set>QDialogButtonBox::Ok</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>DocumentInfoDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>DocumentInfoDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
